low-level/sspl-ll.spec

```diff
@@ -132,9 +132,11 @@ fi
 systemctl stop sspl-ll.service 2> /dev/null || true
 
 %postun
+SSPL_DIR=/opt/seagate/%{product_family}/sspl
 rm -f /etc/polkit-1/rules.d/sspl-ll_dbus_policy.rules
 rm -f /etc/dbus-1/system.d/sspl-ll_dbus_policy.conf
 rm -f /usr/bin/resource_health_view /usr/bin/sspl_bundle_generate /usr/bin/manifest_support_bundle
+rm -f $SSPL_DIR/bin/resource_map
```

Comment:
resource_map is a dir i guess, in which case rm -rf would be needed !

Reply:
done

Reply:
i still see 
> rm -f

Reply:
resource_map inside bin dir is a symlink not a dir.
so keeping it as 

> rm -f $SSPL_DIR/bin/resource_map

Reply:
ok, then its fine